src/neighbors.cc

```diff
@@ -1172,22 +1153,22 @@ neighborUp(const CachePeer * p)
      * for it.
      */
     if (0 == p->n_addresses) {
-        debugs(15, 8, "neighborUp: DOWN (no-ip): " << p->host << " (" << p->in_addr << ")");
+        debugs(15, 8, "DOWN (no-ip): " << *p);
         return 0;
     }
 
     if (p->options.no_query) {
-        debugs(15, 8, "neighborUp: UP (no-query): " << p->host << " (" << p->in_addr << ")");
+        debugs(15, 8, "UP (no-query): " << *p);
         return 1;
     }
 
     if (p->stats.probe_start != 0 &&
             squid_curtime - p->stats.probe_start > Config.Timeout.deadPeer) {
-        debugs(15, 8, "neighborUp: DOWN (dead): " << p->host << " (" << p->in_addr << ")");
+        debugs(15, 8, "DOWN (dead): " << *p);
         return 0;
     }
 
-    debugs(15, 8, "neighborUp: UP: " << p->host << " (" << p->in_addr << ")");
+    debugs(15, 8, "UP: " << *p);
```

Comment:
To avoid creating inconsistent output and/or excessive diff noise, I tried to leave excessive MYNAME prefixes in old debugs() statements unless we modified all such debugs() statements in a function OR the statements already mixed MYNAME and MYNAME-free wording. I am sure I got a couple of cases wrong and will undo or change them upon request. IMO, we should remove (most) such prefixes in a single PR covering all Squid sources.

Reply:
You do not have to change every debugs() in a function. But please do remove the MYNAME portion for debugs() that are touched by this PR already.

Reply:
Done in fbfaaf4 to avoid arguing. Please let me know if I missed any applicable cases.